Resources/FAQ/Jobs-Discord-Asked-Questions.md

```diff
@@ -51,26 +51,56 @@ Yes, CMILib is a library by Zrips that all his plugins require, including Jobs-R
+Jobs-Reborn is open source, and you can view the source code on [Github] (https://Github.com/Zrips/Jobs)
+
+If you want, you can clone it and compile it yourself.
+
+[Find out more here.] (https://Github.com/mrfdev/Jobs/blob/main/Resources/FAQ/Jobs-jar-files.md)
 
 ---
 
-### I have a bug, fix it!
+### I have a bug. Fix it!
+
+Yeah, life is not that easy. Unfortunately, we deal in bug reports, not magic shows. So we will need more information.
+
+First, please check that you're on the latest build of your service engine; type: `/ver` and see how far behind you are.
+
+Update to the latest versions of [CMILib](https://www.spigotmc.org/resources/cmilib.87610/) and [Jobs](https://www.spigotmc.org/resources/jobs-reborn.4216/)
 
-Yeah, life is not that easy, we deal in bug reports, not magic shows. So we will need more information. Please check that you're on the latest build of your service engine; type: `/ver` and see how far behind you are. Additionally, get the latest CMILib jar from SpigotMC.org, and the latest Jobs .jar from SpigotMC.org. Try again. If it still fails, check the latest.log file to see what the errors are during startup and resolve those. If you believe you've found a bug, gather the info about your server and go to Zrips's Github and submit a new issue for review. Explain what the issue is, how to reproduce it. What you've done to mitigate it, what /ver and other versions you have of what you're trying to work with, and include the full error message.
+Try again. If it still fails, check the latest.log file to see the errors during startup and resolve those.
+
+If you believe you've found a bug, gather the info about your server and go to Zrips's Github and submit a new issue for review.
+
+Provide at least the following:
+What the problem is, and how can it be reproduced
+What you've done to mitigate it so far.
+The output of the command /ver
+Versions of other plugins related to your problem
+include the entire error message.
 
 ---
 
 ### Jobs is lagging my server, shows in timings report.
 
-This is a loaded question, because it can mean a lot of things. Another plugin can hang the main thread, forcing a queue to build up or other tasks to wait before they can complete. We will need to know much more information. Including a timings report. Github > new issue > is the best way to deal with that. Think about it first, what info will be helpful to help you the best.
+That's not how the timings work. Jobs showing up as a lagging can mean many things. Another plugin can hang the main thread, forcing a queue to build up or other tasks to wait before completing.
+
+We will need to know much more information, including your timings report.
 
+Going to [Github](https://github.com/mrfdev/Jobs/issues) and starting a new issue is the best way to deal with that.
```

Comment:
This should be pointing to the zrips bug tracker, as this is only for the community documentation